<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Adrian Polyakov</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VKMusic Downloader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/images/logo.ico</normaloff>:/images/logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="2">
     <widget class="QPushButton" name="pause_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;image:url(:/images/pause_button.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSlider" name="volumeSlider">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="sliderPosition">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="login_text">
      <property name="text">
       <string>Номер телефона или электронная почта:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>login</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="7">
     <widget class="QSlider" name="play_status">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QCheckBox" name="saveData">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Сохранить введенные данные?</string>
      </property>
      <property name="text">
       <string>Запомнить</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="progress_label">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Прогресс скачивания:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="stop_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;image:url(:/images/stop_button.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="2">
     <widget class="QLineEdit" name="user_link"/>
    </item>
    <item row="2" column="6">
     <widget class="QToolButton" name="sort_tracks">
      <property name="text">
       <string>Сортировка</string>
      </property>
      <property name="popupMode">
       <enum>QToolButton::InstantPopup</enum>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextOnly</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QLabel" name="link_text">
      <property name="text">
       <string>Ссылка на профиль (пост, альбом) пользователя (сообщества):</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>user_link</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="2">
     <widget class="QLineEdit" name="login"/>
    </item>
    <item row="5" column="5" colspan="2">
     <widget class="QPushButton" name="btnConfirm">
      <property name="toolTip">
       <string>Получить список аудиозаписей пользователя</string>
      </property>
      <property name="text">
       <string>Получить аудиозаписи</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="search">
      <property name="placeholderText">
       <string>Поиск...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="2">
     <widget class="QLineEdit" name="password">
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="7">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tracks_tab">
       <attribute name="title">
        <string>Аудиозаписи</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="trackList">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>artist - name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>name - artist</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="playlists_tab">
       <attribute name="title">
        <string>Плейлисты</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="albumsList">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Плейлист</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="3" column="5" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="format">
       <string>Скачано %v из %m</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLabel" name="password_text">
      <property name="text">
       <string>Пароль:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>password</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>38</height>
    </rect>
   </property>
   <widget class="QMenu" name="music_menu">
    <property name="title">
     <string>&amp;Музыка</string>
    </property>
    <addaction name="saveAll"/>
    <addaction name="saveWithoutLinks"/>
    <addaction name="separator"/>
    <addaction name="downloadAllTracks"/>
    <addaction name="separator"/>
    <addaction name="luckyMe"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>&amp;Помощь</string>
    </property>
    <addaction name="helpDialog"/>
    <addaction name="separator"/>
    <addaction name="aboutDialog"/>
   </widget>
   <addaction name="music_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="saveAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/save_all.png</normaloff>:/images/save_all.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Сохранить</string>
   </property>
   <property name="statusTip">
    <string>Сохранить список аудиозаписей в файл со ссылками для их скачивания</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveWithoutLinks">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить &amp;без ссылок</string>
   </property>
   <property name="statusTip">
    <string>Сохранить список аудиозаписей в файл без ссылок для их скачивания</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="downloadAllTracks">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/download_all.png</normaloff>:/images/download_all.png</iconset>
   </property>
   <property name="text">
    <string>С&amp;качать всё</string>
   </property>
   <property name="statusTip">
    <string>Скачать все аудиозаписи пользователя</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="luckyMe">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Мне повезёт</string>
   </property>
   <property name="statusTip">
    <string>Воспроизвести случайную аудиозапись из списка</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="exit">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Выход</string>
   </property>
   <property name="statusTip">
    <string>Выход из VKMusic Downloader</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="helpDialog">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Помощь</string>
   </property>
   <property name="statusTip">
    <string>Помощь по VkMusic Downloader</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="aboutDialog">
   <property name="text">
    <string>&amp;О программе</string>
   </property>
   <property name="statusTip">
    <string>Информация о VkMusic Downloader</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>login</tabstop>
  <tabstop>password</tabstop>
  <tabstop>user_link</tabstop>
  <tabstop>saveData</tabstop>
  <tabstop>sort_tracks</tabstop>
  <tabstop>btnConfirm</tabstop>
  <tabstop>search</tabstop>
  <tabstop>volumeSlider</tabstop>
  <tabstop>pause_button</tabstop>
  <tabstop>stop_button</tabstop>
  <tabstop>play_status</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>trackList</tabstop>
  <tabstop>albumsList</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
